.local/matrix-runner-2.sh

```
@@ -0,0 +1,201 @@
+
+  local k6pod=""
+  for i in $(seq 1 120); do
+    k6pod=$(kubectl -n redis get pods -l app.kubernetes.io/name=loadgen -o jsonpath='{.items[0].metadata.name}' 2>/dev/null)
+    [ -n "$k6pod" ] && break
+    sleep 1
+  done
+  if [ -z "$k6pod" ]; then
+    echo "  $mult | FAIL: no k6 pod after 120s" | tee -a "$OUT/matrix.log"
+    return
+  fi
+  for i in $(seq 1 150); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)